src/hooks/useAddAdditionalRepositories.js

```diff
@@ -0,0 +1,31 @@
+import { useMutation } from 'react-query';
+
+const additionalRepositoriesMutation = async (data) => {
+  const { keyName, additionalRepos } = data;
+  if (!keyName) {
+    return false;
+  }
```

Comment:
I think we should throw an error if this is called without a keyname. That way `isError` gets set to true instead of silently returning false for the data this endpoint returns